disnake/flags.py

```diff
@@ -823,6 +828,31 @@ def dm_messages(self):
         """
         return 1 << 12
 
+    @flag_value
+    def message_content(self):
+        """:class:`bool`: Whether messages will have access to message content.
+
+        This applies to the following fields on :class:`~disnake.Message` instances:
+
+        - :attr:`~disnake.Message.content`
+        - :attr:`~disnake.Message.embeds`
+        - :attr:`~disnake.Message.attachments`
+        - :attr:`~disnake.Message.components`
+
+        The following cases will always have the above fields:
+
+        - Messages the bot sends
+        - Messages the bot receives in :class:`~disnake.DMChannel` instances
```

Comment:
```suggestion
        - Messages the bot receives as a direct message
```
just to clarify, since you don't always get a `DMChannel` in DMs due to api limitations
(probably the same thing in intents.rst)

Reply:
https://github.com/DisnakeDev/disnake/pull/353/commits/3c3fe5b2ad1b26b9bd0a23afb7462b05ccfd9058